easybuild/scripts/rpath_args.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
```

Comment:
That doesn't work, you can only pass a single argument to a shebang.

And anyway, this doesn't matter much, we call the script with `python rpath_args.py ...` anyway (i.e. with the `python` that is used to run `eb`, i.e. `sys.executable`).